src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -461,4 +461,7 @@
     <value>The '_type' parameter contains unknown resource(s): {0}</value>
     <comment>{0}: Single or list of unknown resources.</comment>
   </data>
+  <data name="IncompatibleSearchParameters" xml:space="preserve">
+    <value>The parameters '{0}' and '{1}' are not compatible.</value>
+  </data>
```

Comment:
Open to different wording - is it worth making it clear that `_summary=false` is permitted?

Reply:
[https://github.com/FirelyTeam/firely-net-sdk/blob/03c257328631716fb335b634396d711e0b0e22eb/src/Hl7.Fhir.Core/Serialization/BaseFhirSerializer.cs#L30](https://github.com/FirelyTeam/firely-net-sdk/blob/03c257328631716fb335b634396d711e0b0e22eb/src/Hl7.Fhir.Core/Serialization/BaseFhirSerializer.cs#L30)

Is there are reason to not just resurface that exception to user?

---
In reply to: [528067067](https://github.com/microsoft/fhir-server/pull/1480#discussion_r528067067) [](ancestors = 528067067)

Reply:
Good question. I originally decided to throw the exception here instead of using this existing exception message thrown by `_fhirJsonSerializer.Serialize()` in `FhirXmlOutputFormatter.cs` or `FhirJsonOutputFormatter.cs` because the `BadRequestException` is automatically wrapped in an `OperationOutcome` if I throw it from `SearchOptionsFactory.cs`.

That being said, I'm not sure if it would actually be ugly to manually add my own `OperationOutcome` in the formatters. Looking into this a bit more now... thanks!

Reply:
On a second thought, it's probably better to wrap this in our exception which would be localizable (hope that's a word). But we can use they wording if you like how they formulate it.

---
In reply to: [528436000](https://github.com/microsoft/fhir-server/pull/1480#discussion_r528436000) [](ancestors = 528436000)

Reply:
Updated the error message in 514a89e.